### PR TITLE
Implemented slack messaging after a deployment

### PR DESCRIPTION
**Pull request by GPaoloni**
Jira: https://bugs.benetech.org/browse/CHI-730

This PR updates `deploy-beta` action to send a Slack message after a deployment.

### CODE REVIEW

File: .github/actions/deploy-beta/action.yml [hunk 1]
```diff
@@ -105,4 +111,13 @@ runs:
...
         TWILIO_AUTH_TOKEN: ${{ inputs.auth-token }}
-      shell: bash
\ No newline at end of file
+      shell: bash
+    # Send Slack notifying success
+    - name: Slack Aselo channel
+      id: slack
+      uses: slackapi/slack-github-action@v1.14.0
+      with:
+        channel-id: "C02L11XCKQU"
```
Comment (nickhurlburt) [inline .github/actions/deploy-beta/action.yml:120]: This doesn't seem all that sensitive, but would it be better as a GH secret?

**Review by nickhurlburt — COMMENTED** [review]
Sweet!

File: .github/actions/deploy-beta/action.yml (hunk 1 above)
Comment (GPaoloni) [reply .github/actions/deploy-beta/action.yml:120]: This is something I have a question about: this channel id is for the channel I've created. Should we use one of the existing ones or create a new one? Then I will move this to be a secret if preferred.

Comment (nickhurlburt) [reply .github/actions/deploy-beta/action.yml:120]: I think having an #aselo-deploys channel like you created is a great idea. Then it doesn't clutter other channels.

**Pull request merged**